src/main/java/pages/LoginPage.java

```diff
@@ -198,20 +200,20 @@ public String getTryDemoCustomerAccessPasswordText() {
     }
 
     public String getCreateAccountButtonFontColorAsHex() {
-        return getColorAsHex(driver, createAccountButton, "color");
+        return getColorAsHex(driver, createAccountButton, CssProperty.COLOR.getPropertyName());
```

Comment:
getColorAsHex should accept CssProperty so it's not needed to get every time propertyName
`public static String getColorAsHex(WebDriver driver, By locator, CssProperty cssProperty){
        WebElement element = driver.findElement(locator);
        String colorRgba = element.getCssValue(cssProperty.getPropertyName());
        return Color.fromString(colorRgba).asHex();
    }`